CHANGES.rst

```diff
@@ -4,7 +4,9 @@ Changelog
 3.0.0a4 (unreleased)
 --------------------
 
-- Nothing changed yet.
+- Fixes #235: Different roles in tests for install and uninstall.
+  Now both is executed as role ``Manager``.
```

Comment:
This confuses me. Are the roles different or are they both Manager? I guess I see what you mean. Maybe change it to this:

Use Manager role in uninstall test.
Then both install and uninstall are executed with role Manager.